app/models/vm_or_template.rb

```diff
@@ -1,4 +1,5 @@
 require 'ancestry'
+require 'addressable'
```

Comment:
Alphabetical ad < an

Reply:
should have been removed now that the caller was removed in https://github.com/ManageIQ/manageiq/pull/21021